File: sortColors.py
def sortColor(nums):

    for num in range(len(nums)):
        min_num = num

        for j in range(len(nums)):
            if (nums[j] > nums[min_num]):
                swap = nums[min_num]
                nums[min_num] = nums[j]
                nums[j] = swap

File: sortColors.js
const sortColors = function(nums) {
      for(let i= nums.length; i>0; i--){
    for(let j=0; j<i-1; j++){
      if(nums[j]>nums[j+1]){
        let temp =nums[j]
                nums[j] = nums[j + 1]
                nums[j + 1] = temp
      }
    }
   }
    
};
